app/assets/javascripts/modules/users/components/new-form.vue

```diff
@@ -89,7 +89,8 @@
     methods: {
       onSubmit() {
         UsersService.save(this.user).then((response) => {
-          const { user, token } = response.data;
+          const { user } = response.data;
+          const token = response.data.plain_token;
```

Comment:
We could rewrite it as 

```js
const { user, plain_token as token } = response.data;
```

but this is okay. Whenever we touch this file again we can change that. No worries.